### ISSUE TITLE
wasting time on [NOPRINT] and mystery period symbols

### ISSUE THREAD

**Issue by cjconroy** [ArctosDB/arctos#8224]
Frequently, when I go to edit, or even look at, an older accession, I hit save and am hit with the error The form cannot be submitted. There are non-printing characters in remarks (or elsewhere). 
These characters were obviously just fine in the past since these are not new accessions. When this happens, the remarks have to be manually clipped out. 
I also have a problem sometimes with the last period in the remarks box. Same error. I then delete the period. hit save. then type another period. hit save and it is fine. 
If Arctos can recognize these invisible non-printing characters, why can't they be deleted or swapped out in bulk somehow?

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Also, if there is a non-printing character in the accession record in accession number, you cannot use the accession until the non-printing character is corrected. In create record it will throw an error whether or not you include the non-printing character.

**Comment by dustymc** [comment]
>just fine

Yes - nonprinting characters are a social issue, they break things like Excel and discoverability but Arctos (the database) has no problem with them internally.

>last period in the remarks box.

I just made a couple, I can't replicate, maybe your browser is somehow being "helpful" (or you habitually type a space after periods??)?

>deleted or swapped out 

Sure, I just need a recipe. I'll move this to internal and pull some data to get started.

> throw an error whether or not you include the non-printing 

Yea, that's not strictly "internal" - it's probably pushing something through javascript, one of the many things that don't play nicely with nonprinting characters.
